src/components/project/ProjectsImportDialog.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <div class="ProjectsLoadDialog">
+    <v-dialog v-model="state.dialog" max-width="1024">
+      <v-card>
+        <v-card-title v-text="'Import projects'" />
+        <v-card-subtitle v-text="'Select a source'" />
+
+        <v-card-text>
+          <v-row class="mb-1">
+            <v-col cols="2">
+              <v-btn-toggle
```

Comment:
This view seems somehow glitched: The icon for the selected source overlaps with the other button and the selected button is not clearly visible any more. Could you check it again?
![grafik](https://user-images.githubusercontent.com/53972736/126981767-41cee669-b6aa-456b-8ca5-4e14cdc5a18b.png)

Reply:
It is strange, in my case it looks clean.

![image](https://user-images.githubusercontent.com/620984/126983871-1a8c07b7-fd5a-40ee-a32b-6c97544e50c4.png)

Reply:
Indeed. I retried it with a clean Firefox Profile, but there were no changes. Did you introduce some changes on your machine, e.g. updates of packages, etc.?

Reply:
I cleaned site data and upgraded packages, but my button group is still smaller  than yours.